src/dom/scroll-render-fix.ts

```diff
@@ -64,11 +64,11 @@ export class ScrollRenderFix {
         e.preventDefault();
         this.raf.read(()=> {
           this.targetY = document.documentElement.scrollTop + e.deltaY;
-        });
-        this.raf.postWrite(()=> {
-            if (this.currentY !== this.targetY) {
-              document.documentElement.scrollTop = this.targetY;
-            }
+          this.raf.postWrite(()=> {
+              if (this.currentY !== this.targetY) {
+                document.documentElement.scrollTop = this.targetY;
```

Comment:
Looks good. Is there a case where e.target would not be the documentElement?

Does this capture a scroll event on a div element with `overflow: scroll`?